src/client.js

```diff
@@ -529,6 +548,44 @@ Client.prototype.clientDetach = function(socketId) {
 	}
 };
 
+Client.prototype.registerPushSubscription = function(session, subscription, noSave) {
+	if (!_.isPlainObject(subscription) || !_.isPlainObject(subscription.keys)
+	|| typeof subscription.endpoint !== "string" || !/^https?:\/\//.test(subscription.endpoint)
+	|| typeof subscription.keys.p256dh !== "string" || typeof subscription.keys.auth !== "string") {
+		session.pushSubscription = null;
+		return;
+	}
+
+	const data = {
+		endpoint: subscription.endpoint,
+		keys: {
+			p256dh: subscription.keys.p256dh,
+			auth: subscription.keys.auth
+		}
+	};
+
+	session.pushSubscription = data;
+
+	if (!noSave) {
+		this.manager.updateUser(this.name, {
+			sessions: this.config.sessions
+		});
+	}
+
+	log.debug(`Subscribed ${this.name} to ${subscription.endpoint}`);
```

Comment:
Do you want to add a `notifications` property to [`debug` in the config file](https://github.com/thelounge/lounge/blob/master/defaults/config.js#L395) or was this just for dev purpose, to remove before merge?